### ISSUE TITLE
Snowflake writer credentials

### ISSUE THREAD

**Issue by ondrejhlavacek** [keboola/kbc-ui#1904]
Když ručně vložím Snowflake credentials do writeru, který dostanu z jinýho writeru nebo z workspace, ukáže se to jako Keboola provided database credentials, ale heslo už nejde zkopírovat.

![image](https://user-images.githubusercontent.com/497675/43891713-53ab8dbe-9bca-11e8-9e69-7c677109d224.png)
 
Super by bylo, kdyby se Keboola provided vs User provided poznávalo nejen podle hostname, ale i podle `#password` a `password` (Keboola provided maj teď uložený nezašifrovaný heslo).

**Comment by ujovlado** [comment]
Jo, to by sme mali fixnut, uz sme to riesili s @kacurez tu ... mozem to potom kuknut

**Comment by kacurez** [comment]
akurat som narazil na rovnaky problem :)
Nedavalo by vacsi zmysel upravit manualne ukladanie aby heslo nesifrovalo ak je z kbc provisioningu(host == keboola.snowflakecomputing.com)? Pretoze stale plati ze to je kbc provisioned db ajked sa jej credentials vlozia rucne ci?
